test/LaunchDarkly.EventSource.Tests/EventSourceTests.cs

```diff
@@ -568,98 +581,6 @@ public async Task When_error_handler_closes_event_source_no_reconnect_attempt_is
             await evt.StartAsync();
 
             Assert.Equal(1, handler.GetRequests().Count());
-        }
-
-        public async Task Connection_closed_before_threshold_gets_increasing_backoff_delay()

```

Comment:
Unrelated to this PR: I removed this test and the following test because they were fatally flawed and guaranteed to be unstable. The only reason we weren't seeing frequent intermittent failures on them in CI was that... they weren't actually being run at all, due to the lack of a `[Fact]` annotation.

The problem is that these tests were trying to measure the timing of actions being done in a live HTTP session between client and server. Using actual I/O makes the execution time completely unpredictable; for that matter, even without I/O, task scheduling in .NET is unpredictable enough that even a simple test involving multiple tasks will have wildly variable execution time.

So, while we of course can and do test the lower-level `ExponentialBackoffWithDecorrelation` logic separately, if we really want to verify that `EventSource` is _using_ the backoff logic as intended, we would need to add some more abstractions/instrumentation inside `EventSource` to make it possible to observe what it _intends_ the backoff delays to be, rather than what they really end up being.